src/test/java/org/mifos/integrationtest/cucumber/stepdef/VoucherManagementStepDef.java

```diff
@@ -310,6 +310,21 @@ public void createAndActivateVoucher(){
         iCanCreateAnVoucherRequestDTOForVoucherActivation();
         iCallTheActivateVoucherAPIWithExpectedStatusOfAndStub(202, "/activateVoucher");
     }
+    @Before("@createVoucher")
+    public void createVoucher(){
+        iCreateAnIdentityMapperDTOForRegisterBeneficiary();
+        mockServerStepDef.checkIfMockServerIsInjected();
+        mockServerStepDef.startMockServer();
+        mockServerStepDef.startStub("/createVoucher", PUT, 200);
+        mockServerStepDef.startStub("/activateVoucher", PUT, 200);
+        iCallTheVoucherCreateAPIWithExpectedStatusOf(202, "/createVoucher");
```

Comment:
Are we calling this "createVoucher" API on the voucher service? the API doesn't follow naming conventions.

Reply:
this is api stub for callback